qemu/tests/cfg/virtio_fs_share_data.cfg

```diff
@@ -95,7 +95,7 @@
             no Host_RHEL.m8
             fs_binary_extra_options += " --log-level debug"
             winapi_dir_name = test_winapi
-            create_dir_winapi_cmd = python -c "import ctypes; dll=ctypes.windll.kernel32;dll.CreateDirectoryA('%s\\${winapi_dir_name}', 0)"
+            create_dir_winapi_cmd = python -c "import ctypes; kernel32 = ctypes.WinDLL('kernel32', use_last_error=True); kernel32.CreateDirectoryW('%s\\${winapi_dir_name}', 0)"
```

Comment:
What about the old pattern, do we still need to consider the compatilbility of the python version lower than python3? I mean that here we can not delete the old patten directly, but to make it compatible for both old and current python, right?

Reply:
The new cmd is compatible for both.

Reply:
Ah get it. Thanks.